packages/cli/src/util/env/get-env-records.ts

```diff
@@ -3,10 +3,19 @@ import Client from '../client';
 import { ProjectEnvVariable, ProjectEnvTarget } from '../../types';
 import { URLSearchParams } from 'url';
 
+/** The CLI command that was used that needs the environment variables. */
+export type EnvRecordsSource =
+  | 'vercel-cli:env:ls'
+  | 'vercel-cli:env:add'
+  | 'vercel-cli:env:rm'
```

Comment:
What about `vercel-cli:env:pull`?

That is a different command than `vercel-cli:pull`

Reply:
Thanks for catching this! Adding that.